src/onboarding/registration/PictureInput.tsx

```diff
@@ -108,34 +112,24 @@ function PictureInput({ picture, onPhotoChosen, backgroundColor }: Props) {
 
 const styles = StyleSheet.create({
   container: {
-    width: 120,
-    height: 120,
-    borderRadius: 60,
+    width: PICTURE_SIZE,
+    height: PICTURE_SIZE,
+    borderRadius: PICTURE_SIZE / 2,
     alignSelf: 'center',
     justifyContent: 'center',
     alignItems: 'center',
   },
-  photoIconContainer: {
-    width: 32,
-    height: 32,
+  editIconContainer: {
+    width: 40,
+    height: 40,
     position: 'absolute',
-    right: 0,
-    bottom: 0,
-    borderRadius: 16,
-    backgroundColor: colors.greenUI,
+    left: 37,
```

Comment:
should this be `PICTURE_SIZE / 2`? or is it unrelated?

Reply:
It's unrelated

Reply:
omg my basic math skills completely failed me :facepalm: